builtin/logical/database/backend.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	"github.com/hashicorp/errwrap"
 	log "github.com/hashicorp/go-hclog"
-	"github.com/hashicorp/go-uuid"
+	uuid "github.com/hashicorp/go-uuid"
```

Comment:
you wish to operate after depositing, please transfer the funds from "My Wallet" to the corresponding account, and select "Transfer" to complete the process